Sources/TinyNetworking/Endpoint.swift

```diff
@@ -60,10 +60,17 @@ public struct Endpoint<A> {
     ///   - query: query parameters to append to the url
     ///   - parse: this converts a response into an `A`.
     public init(_ method: Method, url: URL, accept: ContentType? = nil, contentType: ContentType? = nil, body: Data? = nil, headers: [String:String] = [:], expectedStatusCode: @escaping (Int) -> Bool = expected200to300, timeOutInterval: TimeInterval = 10, query: [String:String] = [:], parse: @escaping (Data?, URLResponse?) -> Result<A, Error>) {
-        var comps = URLComponents(string: url.absoluteString)!
-        comps.queryItems = comps.queryItems ?? []
-        comps.queryItems!.append(contentsOf: query.map { URLQueryItem(name: $0.0, value: $0.1) })
-        request = URLRequest(url: comps.url!)
+        var requestUrl : URL
+        if query.isEmpty {
+            requestUrl = url
+        }
+        else {
```

Comment:
in our code style (which we didn't publish anywhere, so you couldn't know) we write the closing brace and else on the same line. Do you mind changing that?

Reply:
Fixed :)

btw, maybe it would be nice to use 'swiftformat' for the project, then such rules could be applied automatically and other formatting things like trailing whitespace fixed automatically? Like:

```
swiftformat --elseposition same-line .
```